test/global-table.test.ts

```diff
@@ -0,0 +1,41 @@
+import { StackProps, aws_dynamodb } from 'aws-cdk-lib';
+import * as cdk from 'aws-cdk-lib';
+import { Template } from 'aws-cdk-lib/assertions';
+import * as ddb from 'aws-cdk-lib/aws-dynamodb';
+import { GlobalTable } from '../src/global-table';
+
+test('Create global table with partition key and default properties', () => {
+  const region = 'us-east-1';
+  const app = new cdk.App();
+  const newstack = new cdk.Stack(app, 'id', {
+    env: { region: region },
+  });
+  new GlobalTable(newstack, 'hello', {
+    partitionKey:
+    {
+      name: 'id',
+      type: ddb.AttributeType.STRING,
+    },
+  });
```

Comment:
Indentation style still needs to be updated to the compact version, see this comment: https://github.com/cdklabs/awscdk-dynamodb-global-tables/pull/1#discussion_r941871035